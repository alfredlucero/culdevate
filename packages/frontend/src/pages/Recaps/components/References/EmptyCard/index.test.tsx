import React from "react";
import { render } from "@testing-library/react";
import ReferencesEmptyCard from "./index";

describe("<ReferencesEmptyCard />", () => {
  test("should render without error", () => {
    const { container } = render(
      <ReferencesEmptyCard onClickAdd={() => {}} className="extra-list-class" testId="testId" />,
    );

    expect(container).toMatchSnapshot();
  });
});
