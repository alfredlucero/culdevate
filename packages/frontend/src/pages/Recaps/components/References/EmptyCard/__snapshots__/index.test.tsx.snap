// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReferencesEmptyCard /> should render without error 1`] = `
<div>
  <div
    class="bg-white shadow-md rounded px-8 py-6 h-64 p-4 flex flex-col justify-center items-center extra-list-class"
    data-testid="testId"
  >
    <div
      class="mb-6 flex justify-center"
    >
      <div
        class="flex justify-center items-center p-3 rounded-lg bg-gray-200 recap-icon"
      >
        <div
          class="font-sans text-3xl text-teal-400"
          id=""
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-address-card fa-w-18 "
            data-icon="address-card"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M528 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h480c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-352 96c35.3 0 64 28.7 64 64s-28.7 64-64 64-64-28.7-64-64 28.7-64 64-64zm112 236.8c0 10.6-10 19.2-22.4 19.2H86.4C74 384 64 375.4 64 364.8v-19.2c0-31.8 30.1-57.6 67.2-57.6h5c12.3 5.1 25.7 8 39.8 8s27.6-2.9 39.8-8h5c37.1 0 67.2 25.8 67.2 57.6v19.2zM512 312c0 4.4-3.6 8-8 8H360c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16zm0-64c0 4.4-3.6 8-8 8H360c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16zm0-64c0 4.4-3.6 8-8 8H360c-4.4 0-8-3.6-8-8v-16c0-4.4 3.6-8 8-8h144c4.4 0 8 3.6 8 8v16z"
              fill="currentColor"
            />
          </svg>
        </div>
      </div>
    </div>
    <h4
      class="font-mono font-bold text-gray-800 text-xl mb-4 text-center"
    >
      References
    </h4>
    <p
      class="font-sans text-gray-800 text-base mb-6"
    >
      Recap the people you have worked with and who can vouch for you in your next move.
    </p>
    <div
      class="flex justify-center"
    >
      <button
        class="font-bold py-2 px-4 rounded font-bold py-2 px-4 rounded bg-teal-500 text-white hover:bg-teal-700 "
        type="button"
      >
        <div
          class="flex items-center"
        >
          Add a Recap
        </div>
      </button>
    </div>
  </div>
</div>
`;
